server/libs/job/src/interfaces/asset-uploaded.interface.ts

```diff
@@ -10,9 +10,4 @@ export interface IAssetUploadedJob {
    * Original file name
    */
   fileName: string;
-
-  /**
-   * File size in byte
-   */
-  fileSize: number;
```

Comment:
The `fileSize` come as a property of the uploaded file, so we don't have to user another mechanism to calculate the file size. Since the info is readibly available, it is conveninent to insert to the EXIF table.